### ISSUE TITLE
Attribute deserialization failed: Unexpected character encountered while parsing value:

### ISSUE THREAD

**Issue by mwantia** [gpailler/MegaApiClient#240]
I have two accounts: 
- The firstOne I my "main" account, which I privately use to upload all my stuff to the cloud.
- The second one is an account I mainly just created to further test this repository, so it only contains 3 files and 1 folder.

I created a outgoing share from the second account to my main account and gave myself full access, so I can upload and download to the folder even from the first account.
If I upload a new file, an image for example I can see and access it from my browser without any problems but whenever I use the repository, login to my second account and load the nodes I receive the following errored node:
![image](https://github.com/gpailler/MegaApiClient/assets/59731589/69ef8fe1-4253-4b8e-a811-28775faa4cb5)

It seems that this problem was also already mentioned in another Issue #231, with a possible explaination for the problem.

**Comment by gpailler** [comment]
Yes it's a known issue with the library. Unfortunately, I don't have a lot of time to allocate to MegaApiClient to investigate and fix this part. It would be great if you can improve this part and propose a PR.
